assembl/fabfile.py

```diff
@@ -3296,45 +3304,56 @@ def secure_sshd_fail2ban():
 
 
 @task
-def ensure_private_configs():
+def migrate_important_information(path, to=None):
     """
-    Private configs are created once for a server. During CI/CD, we want these configurations
-    to survive each version upgrade
+    Move important files that are meant to survive moving to an automated deployment strategy
     """
-    remote_random_path = os.path.join(env.projectpath, 'random.ini')
-    private_config_path = normpath(os.path. join(env.projectpath, '..', 'global_configs'))
-    if exists(remote_random_path):
-        run('mkdir -p %s' % private_config_path)
-
+    if exists(path):
         # Ensure that the server's random.ini survives (under migration). New servers will generate own random.ini
-        remote_random_path_destination = normpath(os.path.join(private_config_path, 'random.ini'))
+        basename = os.path.basename(path)
+        remote_random_path_destination = normpath(to, basename)
```

Comment:
variable should be renamed as it doesn't refer to random.ini anymore, and above comment should also be removed.